src/SamplesApp/SamplesApp.Droid/Properties/AndroidManifest.xml

```diff
@@ -2,4 +2,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="SamplesApp" android:versionCode="1" android:versionName="1.0">
   <uses-sdk android:minSdkVersion="16"  android:targetSdkVersion="28"/>
   <application android:label="SamplesApp"></application>
-</manifest>
\ No newline at end of file
+  <meta-data android:name="com.google.android.geo.API_KEY" android:value="YOUR API KEY"/>
```

Comment:
We probably want an actual API key, should discuss with @MatFillion

Reply:
we discussed already, with @jeromelaban and we won't put an API Key, ideally it would need to be set by the build and kept secret. For now you need to input your own key to test.